hexrd/ui/constants.py

```diff
@@ -71,3 +71,14 @@ class OverlayType(Enum):
 PAN = 'pan/zoom'
 
 DOCUMENTATION_URL = 'https://hexrd-canonical.readthedocs.io/'
+
+KNOWN_HDF5_PATHS = [
+    ['ATTRIBUTES/TARGET_ORIENTED_IMAGE/DATA', 'DATA'],
+    ['ATTRIBUTES/PINHOLE_ORIENTED_IMAGE/DATA', 'DATA'],
+    ['ATTRIBUTES/PSL_FADE_CORR_IMAGE/DATA', 'DATA'],
+    ['ATTRIBUTES/HDR_IMAGE/DATA', 'DATA'],
+    ['ATTRIBUTES/CORR_IMAGE/DATA', 'DATA'],
+    ['ATTRIBUTES/TIME_ADJUSTED_IMAGE/DATA', 'DATA'],
+    ['ATTRIBUTES/PSL_IMAGE/DATA', 'DATA'],
+    ['DATA', 'DATA'],
+]
```

Comment:
Yes, this was more so the kind of thing I was thinking of :slightly_smiling_face: